src/bin/languageserver/mod.rs

```diff
@@ -1761,7 +1765,9 @@ impl<'a> Builder<'a> {
                         .filter(|h| h.0 == i)
                         .map(|(_, i)| {
                             let mut i = i.clone();
-                            i.val.def_path = defs_to_files[&i.val.def_type].clone();
+                            if let Some(path) = defs_to_files.get(&i.val.def_type) {
+                                i.val.def_path = path.clone();
+                            }
                             i
```

Comment:
There's one more place where `defs_to_files` is used. https://github.com/hyperledger/solang/blob/main/src/bin/languageserver/mod.rs#L1722

As part of https://github.com/hyperledger/solang/pull/1560, 
I have made similar changes (for example, https://github.com/hyperledger/solang/pull/1560/files#diff-bf2200d9cb4eee660f8ff53c8416979a1f0ebe5d5856609e7d7f46fe0c8e9846R1935).